### DISCUSSION TITLE
perguntas: repositorio do Readme.md sumiu do meu perfil

### DISCUSSION THREAD

**Discussion by marehead** [marehead/marehead#4] [Q&A] [answered]
como recuperar readme.md do deashboard que sumiu troquei somente o readme.md para profile.md como faço para recuperar o ultimo save?

**Comment by toptoptuns** [comment]
Sim tem como recuperar !

**Reply by marehead** [reply]
como?

**Comment by toptoptuns** [comment, marked as answer]
basta ir em seu repositorio readme.md que vc renomeou para profile.md  e nomear de volta para README.md

**Reply by marehead** [reply]
thanks

**Reply by toptoptuns** [reply]
Respondido?

**Reply by marehead** [reply]
vlw
